lib/mpl_toolkits/axes_grid1/parasite_axes.py

```diff
@@ -232,9 +220,9 @@ def twiny(self, axes_class=None):
 
         parasite_axes_class = parasite_axes_class_factory(axes_class)
 
-        ax2 = parasite_axes_class(self, sharey=self, frameon=False)
+        ax2 = parasite_axes_class(self, sharey=self)
         self.parasites.append(ax2)
-        ax2._remove_method = self._remove_twiny
```

Comment:
Doesn't this mean we can get rid of `_remove_twiny` and `_remove_twinx`?

Reply:
indeed